draft/root-transitions.md

```diff
@@ -0,0 +1,54 @@
+---
+RFC: XXXX
+Author: Kyle Fuller
+Status: Draft
+Created: 2015-09-21
+Last Modified: 2015-09-21
+---
+
+# API Blueprint RFC XXXX: Root Transitions
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Proposed Syntax](#proposed-syntax)
+
+## Abstract
+
+This RFC proposes that you can define the transitions you may make from the
+root of an API described by API Blueprint.
+
+## Motivation
+
+There is currently no way to express transitions from the root of an API.
+You can express transitions from a resource using the relation keyword, but it
```

Comment:
> transitions from a resource

Did you mean to say `transitions of a resource`? Because otherwise I don't understand what you mean by this sentence. Even more especially if you take the sentence above this line into mind.